MeshLib/Location.h

```diff
@@ -20,6 +20,14 @@ namespace MeshLib
 
 enum class MeshItemType { Node, Edge, Face, Cell, IntegrationPoint };
 
+/// Returns a string output for a specific error flag
```

Comment:
`MeshItemType` instead of error flag? Maybe the spelling should be the same as in the enum, i.e., CamelCase.